spec/models/calculator_resolver_spec.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe CalculatorResolver, type: :model do
+  subject { CalculatorResolver }
+  let(:calculator) { build(:calculator) }
+  let!(:calculation) { create(:calculation, value: 'P1 * P2 / P3', type: 'Calculation', label: 'one', kind: 'result', calculator: calculator) }
+  let!(:value1) { create(:value, value: 'Value', type: 'Value', label: 'two', kind: 'result', calculator: calculator) }
+  let!(:value2) { create(:value, value: 'Value', type: 'Value', label: 'three', kind: 'parameter', calculator: calculator) }
+  let!(:value3) { create(:value, value: 'Value', type: 'Value', label: 'four', kind: 'parameter', calculator: calculator) }
+  let!(:value4) { create(:value, value: 'Value', type: 'Value', label: 'five', kind: 'parameter', calculator: calculator) }
+
+  it { is_expected.to respond_to(:call) }
```

Comment:
You have tests for call below

Reply:
Thank you. I've removed this duplication.